portfolio/src/main/webapp/script.js

```diff
@@ -34,13 +32,16 @@ async function getMarcoStats() {
   // reference to create HTML.
   const stats = await responseFromServer.json();
 
-  const statsListElement = document.getElementById('function-container');
+  // Obtains ID "greeting-container" from HTML file
+  const statsListElement = document.getElementById('greeting-container');
   statsListElement.innerHTML = '';
 
+  // Selects a random element in the ArrayList and prints it in the greeting-container
+  const index = Math.floor(Math.random() * 3);
```

Comment:
3 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) here, and will need to change to stay in sync with the back-end data source. I think you should be able to replace it with `stats.length` to make this code future-proof and improve readability.